modules/client/src/tokenVoting/internal/client/encoding.ts

```diff
@@ -55,7 +58,7 @@ export class TokenVotingClientEncoding extends ClientCore
       args,
     );
     return {
-      id: TOKEN_VOTING_PLUGIN_ID,
+      id: LIVE_CONTRACTS[network].tokenVotingRepo,
```

Comment:
Typescript checks the type, but we might want to have some basic validation that `LIVE_CONTRACTS[network]` actually has something